packages/@sanity/core/src/commands/dataset/copyDatasetCommand.js

```diff
@@ -103,33 +143,19 @@ export default {
         `Copying dataset ${chalk.green(sourceDatasetName)} to ${chalk.green(targetDatasetName)}...`
       )
 
-      if (!flags.wait) {
+      if (flags.background) {
+        /*
+         * Use doesn't want to wait for copy to finish.
```

Comment:
Typo: Should be `User`.

Reply:
Also, not sure to have a three-line comment when the explanation of what this flag does is in the top of this file.

Reply:
Good point! 🙂 Removed the comment in ae19b5957